docs/topics/db/fixtures.txt

```diff
@@ -126,11 +126,31 @@ You could also write a decorator to encapsulate this logic::
 Just be aware that this logic will disable the signals whenever fixtures are
 deserialized, not just during ``loaddata``.
 
-Note that the order in which fixture files are processed is undefined. However,
-all fixture data is installed as a single transaction, so data in
-one fixture can reference data in another fixture. If the database backend
-supports row-level constraints, these constraints will be checked at the
-end of the transaction.
+Fixtures can be loaded in a specified order. For example:
```

Comment:
I think we need to out this in a subsection, with a title along the lines `In which order are the fixtures loaded?` (I'm not sure I like all the title-questions, but my proposal would be consistent with the existing style) @evildmp any opinion on the "titles as questions"?

Reply:
@dtraleigh Is this is something you could work on? I do think we should have the "order of fixtures" in its own subsection now that we are expanding it.

Reply:
@nessita How does this subsection sound to you?